## zaptel_perl by Tzafrir Cohen <tzafrir.cohen@xorcom.com>
##
## Point Zaptel PERL modules to the right directory under
## which to install perl modules.
## (sitelib in Debian is under /usr/local. We don't want to install
## there)

Index: dahdi-tools-2.3.0/xpp/Makefile
===================================================================
--- dahdi-tools-2.3.0.orig/xpp/Makefile	2010-03-07 08:45:12.000000000 +0100
+++ dahdi-tools-2.3.0/xpp/Makefile	2010-04-23 16:02:33.000000000 +0200
@@ -28,7 +28,7 @@
 DATADIR		= $(datadir)/dahdi
 MANDIR		= $(mandir)/man8
 HOTPLUG_USB_DIR	= $(sysconfdir)/hotplug/usb
-PERLLIBDIR	:= $(shell eval `perl -V:sitelib`; echo "$$sitelib")
+PERLLIBDIR	:= $(shell eval `perl -V:vendorlib`; echo "$$vendorlib")
 PERL_DIRS	:= $(shell cd perl_modules; find * -name '[A-Z]*' -type d| xargs)
 PERL_MODS_PAT	:= *.pm $(PERL_DIRS:%=%/*.pm)
 PERL_MODS	:= $(shell cd perl_modules; echo $(PERL_MODS_PAT))
